### ISSUE TITLE
Removing all tags prevents adding new ones afterwards

### ISSUE THREAD

**Issue by openmedi** [willsALMANJ/Zutilo#50]
The title pretty much sums it up: Under OS X 10.11.2 and in Zotero Standalone 4.0.28.8 with Zutilo 1.3.0 and by using a previously set keyboard shortcut (`cmd+alt+ctrl+shift+e`; but that is probably not it…) Zutilo initially does what it should and removes all Tags from an entry. Alas, there's problems when I afterwards want to add new tags: I can start to add them, meaning, I can add a new tag, fill in the title field of the new tag, but when I press return, the tag appears to have never been created in the first place.

EDIT: I should've added, that a workaround for the bug is to restart Zotero.

**Comment by willsALMANJ** [comment]
Hmm, that's tricky. I tested with 4.0.28 on Linux and could not reproduce this. Are you just trying out this function or did it work before a recent update to Zotero? I can try to find an OS X machine to test this on, though I don't see why this operation would depend on the operating system.

**Comment by openmedi** [comment]
I was just trying it out for the first time and thought it might be a good way to get rid of all the automatic tags, that were generated before I knew how to turn that function off in Zotero.

**Comment by willsALMANJ** [comment]
Unfortunately, my old OS X laptop is dead, so I haven't been able to test this on OS X. On Linux, I don't see the problem. Maybe there is more information in the Browser Console (see e.g. https://github.com/willsALMANJ/Zutilo/blob/master/i18n/en-US/readme/docs/BUGS.md). Otherwise, I think I have to just leave this issue open until someone can give more insight on what's going on since I can't see it myself.

**Comment by Jdodo45** [comment]
I confirm the bug. On windows 7 with Zutilo 2.0.2 in firefow 50.1.0 .

Same as Openmedi : I get rid of all the automatic tags (using right click context menu). But no way to add tags after that. (I did not tried in the standalone).

ps: and thanks so much for your awesome plugin, I could not survive without it!

**Comment by WiliTest** [comment]
I confirm the bug. On windows 7 with Zutilo 2.0.2 in firefow 50.1.0 .
Same as Openmedi : I get rid of all the automatic tags (using right click context menu). But no way to add tags after that. (I did not tried in the standalone).
ps: and thanks so much for your awesome plugin, I would have not survived without it!

**Comment by willsALMANJ** [comment]
Thanks for the confirmation of the bug. I tried saving that page to Zotero and then removing the tags a couple times, but I was still able to add tags each time afterwards. I think we have to leave the issue open for now until we find some way to reliably trigger the behavior so we can debug what is going on.

The remove tags function basically just calls the Zotero item's `removeAllTags()` method, so there is not much Zutilo could be doing wrong. A lot of the backend was changed in Zotero version 5, so maybe this problem will not occur there.
